packages/core/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const lernaAliases = require('lerna-alias').jest()
+
+module.exports = {
+  testEnvironment: 'node',
+  moduleNameMapper: lernaAliases,
+  transform: {
```

Comment:
this is kinda a temporary hack for now, we should be able to just use `babel-jest`, but we'd have to install https://github.com/babel/babel-bridge and cant do it right now because we test babel plugin with both babel@6 and babel@7

Reply:
after tomorrow's babel stable release we can start supporting only 7 version